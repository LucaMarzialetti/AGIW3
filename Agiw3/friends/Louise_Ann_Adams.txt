--url_fb https://www.facebook.com/louise.annadams
--friends
VM28307:3 Chris Taffy Baldwin
VM28307:3 Monkeyboy Guinness
VM28307:3 Barry Buck
VM28307:3 Aaron Reeves
VM28307:3 Del Boy Trotter
VM28307:3 Aaron Foreman
VM28307:3 Martin Simpson
VM28307:3 Felix Charles
VM28307:3 Daniel Dk Haines
VM28307:3 Bill Bo Bonkers
VM28307:3 Tawnya Webb
VM28307:3 Steve Waters
VM28307:3 Callum Street
VM28307:3 Michael Brosnan
VM28307:3 Steven Garren Coleman
VM28307:3 Todd Higer
VM28307:3 Jamie Hall