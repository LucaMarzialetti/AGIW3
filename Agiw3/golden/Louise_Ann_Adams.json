{
    "PATTERN": {
        "tel": ["800-562-8860"],
        "addr": [],
        "email": []
    },
    "NER": {
        "LOC": ["Beautiful Blossom"],
        "ORG": [
            "Lauren G. Adams LLC",
            "Facebook"
        ],
        "PER": ["Lauren G Adams Lauren G Adams Lauren"]
    },
    "url": "http://www.laurengadams.com/"
}